src/core/bot/transit.py

```
@@ -0,0 +1,180 @@
+                    return MessageID(module, cmd)
+
+        # 如果是频道无at消息可能是发错了或者并非用户希望的处理对象
+        if message.is_guild_public():
+            return MessageID("default.manual", "no_reply")
+
+        if '/' in func:
+            return MessageID("default.manual", "reply_not_implemented")
+        else:
+            return MessageID("default.manual", "reply_key_words_func")
+
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)